### ISSUE TITLE
Comparative method's batch size.

### ISSUE THREAD

**Issue by JasonYpro** [zizheng-guo/RhythmMamba#7]
Hello, may I ask if you have kept the batch size and learning rate consistent when comparing with various methods? For example, I see in your configuration file that the batch size for DeepPhys, PhysNet, TS-CAN, and PhysFormer is 4, while the batch size for RhythmMamba is 16.

**Comment by zizheng-guo** [comment]
If RhythmMamba is maintained with the default configuration, the training process is difficult to converge. By reducing the learning rate and increasing the batch size, the convergence of the training process can be achieved. 
And you can try to adjust the batchsize and learning rate of other methods, with high probability of little change in performance (within randomness).
